templates/ui-app/csharp/Runner.csproj

```
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Tizen.NET.Sdk/1.1.5">
```

the latest version of Tizen.NET.Sdk is 1.1.6. 
```suggestion
﻿<Project Sdk="Tizen.NET.Sdk/1.1.6">
```

---

A warning will be generated if the app project's SDK version mismatches that of `Tizen.Flutter.Embedding.csproj`.

```
/home/swift/Git/flutter-tizen/embedding/csharp/Tizen.Flutter.Embedding.csproj : warning MSB4240: Multiple versions of the same SDK
"Tizen.NET.Sdk" cannot be specified. The previously resolved SDK version "1.1.5" from location
```

It seems it's still safe and I'll make the change in all csproj files.